src/lua/compat.lua

```diff
@@ -96,6 +103,12 @@ local options = {
         brief = SQL_SEQ_SCAN_DEFAULT_BRIEF,
         action = tweak_action('sql_seq_scan_default', true, false),
     },
+    box_tuple_new_arg_list = {
+        default = 'new',
+        obsolete = nil,
+        brief = BOX_TUPLE_NEW_ARG_LIST_BRIEF,
+        action = tweak_action('box_tuple_new_arg_list', true, false)
+    },
```

Comment:
Please, introduce the breaking change in a separate commit.

Reply:
Added a separate commit.